.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Configures dependabot
+
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: '/'
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "chore: "
```

Comment:
Note sure if we want to prefix the commit message or just use default.

Example commit with prefix: https://github.com/GeckoEidechse/mdbook-gitbook/pull/22/commits/67214ac62ce3defbd780c1378d95471c1eba7c5a

Reply:
cc @catornot

Reply:
ig it could just be `chore: `

Reply:
That's what it is rn :P